database_sanitizer/sanitizers/user.py

```diff
@@ -12,7 +12,7 @@ def sanitize_email(value):
     given_name = given_names[num1 % given_names_count]
     surname = surnames[num2 % surnames_count]
     case_convert = (text_type.lower if num3 % 8 > 0 else lambda x: x)
-    return '{first}.{last}@x{num:x}.sanitized.net'.format(
+    return '{first}.{last}@x{num:x}.sanitized.invalid'.format(
```

Comment:
I don't think it's a good idea to use the `.invalid` TLD, since the application logic might consider such e-mail addresses as invalid, e.g. raise ValidationErrors on the forms.

Reply:
Hmm.. Good point. Perhaps `san.example.com`? To keep some semantics of the "sanitized", but to avoid having it too long?

Reply:
Sounds good.

Reply:
Updated.